ledger/src/blockstore.rs

```diff
@@ -158,6 +161,7 @@ pub struct Blockstore {
     block_height_cf: LedgerColumn<cf::BlockHeight>,
     program_costs_cf: LedgerColumn<cf::ProgramCosts>,
     bank_hash_cf: LedgerColumn<cf::BankHash>,
+    optimistic_slot_cf: LedgerColumn<cf::OptimisticSlot>,
```

Comment:
`optimistic_slot_cf` -> `optimistic_slots_cf`,  `OptimisticSlot` -> `OptimisticSlots`.

Reply:
There's a mix of singular and plural in the `_cf` naming. I'd prefer to leave this as-is.

Reply:
Yep, that's bad as we are essentially storing more than one entry instead of just one. For slots, since we have other slot-based column families such as `DuplicateSlots` and `DeadSlots`, I think it's better that we keep the same naming for similar types of data.

Reply:
I'll change this to match the other "Slots" column families.